src/frontends/pytorch/src/op/unfold.cpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_unfold(NodeContext& context) {
+    // constants
+    auto const_0 = context.mark_node(opset10::Constant::create(element::i64, Shape{}, {0}));
+    auto const_1 = context.mark_node(opset10::Constant::create(element::i64, Shape{}, {1}));
+    auto const_0_list = context.mark_node(opset10::Constant::create(element::i64, Shape{1}, {0}));
+    auto const_1_list = context.mark_node(opset10::Constant::create(element::i64, Shape{1}, {1}));
+    auto const_neg_1_list = context.mark_node(opset10::Constant::create(element::i64, Shape{1}, {-1}));
+
+    // get inputs and prepare auxiliary nodes
+    auto input = context.get_input(0);
+    auto sizes = context.mark_node(std::make_shared<opset10::ShapeOf>(input));
+    auto input_rank = context.mark_node(std::make_shared<opset10::ShapeOf>(sizes));
+
+    auto dimension =
+        context.mark_node(opset10::Constant::create(element::i64, Shape{1}, {context.const_input<int64_t>(1)}));
+    auto dimension_plus_1 = context.mark_node(std::make_shared<opset10::Add>(dimension, const_1_list));
+
+    int size_int = context.const_input<int64_t>(2);
+    auto size_scalar = context.mark_node(opset10::Constant::create(element::i64, Shape{}, {size_int}));
```

Comment:
```suggestion
    auto size_scalar = context.mark_node(Constant::create(element::i64, Shape{}, {size_int}));
```
can simply like this due to using namespace

Reply:
done